interface/wx/string.h

```diff
@@ -915,7 +915,7 @@ class wxString
         @see Cmp(), IsSameAs().
     */
     int CmpNoCase(const wxString& s) const;
-
+    
```

Comment:
The changes in this file is an accidental whitespace change and as such should be reverted,  so the file is not included in the eventual final commit.